cross/epic5/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = epic5
+PKG_VERS = 1.4
+PKG_EXT = tar.xz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://ftp.epicsol.org/pub/epic/EPIC5-PRODUCTION
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/openssl cross/ncurses cross/libiconv
+
+HOMEPAGE = http://www.epicsol.org/
+COMMENT  = Internet Relay Chat client for UNIX like systems
+LICENSE  = BSD3
+
+GNU_CONFIGURE = 1
+CONFIGURE_ARGS = --with-ipv6 --with-ssl --with-wserv --with-iconv
```

Comment:
Add `ac_cv_func_getpgrp_void=yes` to CONFIGURE_ARGS here. It should work, if not, there is something wrong elsewhere.
See other cross packages for example, there are other with configure checks overrides.